docs/faq.md

```diff
@@ -242,3 +242,12 @@ RHCOS inherits the majority of its configuration from Fedora CoreOS, so we aim t
 If the package makes sense to include in Fedora CoreOS, it will ultimately be included in RHCOS in the future when the [fedora-coreos-config](https://github.com/coreos/fedora-coreos-config) submodule is updated in this repo.
 
 If the package is **not** included in Fedora CoreOS, you may submit a PR to this repo asking for the inclusion of the package with the reasoning for including it.
+
+## Q: How do I replace the current Kernel with Kernel-RT or a new Kernel version in RHCOS?
```

Comment:
I think the instructions below are only for the first case (moving from kernel to kernel-rt). For changing kernel versions (but not type), `rpm-ostree override replace [list of RPMs]` should be enough.

Reply:
A great! Thanks for reviewing it @jlebon